packages/dom/test/unit/lib/ElementAssertion.test.tsx

```diff
@@ -42,4 +53,81 @@ describe("[Unit] ElementAssertion.test.ts", () => {
       });
     });
   });
+
+  describe(".toContainElement", () => {
+    context("when the ancestor element contains the descendant element", () => {
+      it("returns the assertion instance", () => {
+        const { getByTestId } = render(<ContainsTestComponent/>);
+
+        const grandparent = getByTestId("grandparent");
```

Comment:
You can use here async/await functions eg: ` const grandparent =  await getByTestId("grandparent");`, just to ensure that asynchronous operations complete before assertions, that also prevents false positive/negatives due unresolved promises.

Reply:
thanks team!